packages/server/src/integrations/base/sql.ts

```diff
@@ -648,6 +686,18 @@ class SqlQueryBuilder extends SqlTableQueryBuilder {
     }
     return results.length ? results : [{ [operation.toLowerCase()]: true }]
   }
+
+  log(query: string, values?: any[]) {
```

Comment:
How concerned should we be here about logging sensitive customer information?

Reply:
This won't be turned on by default - it is functionality that will mainly be used in self host environments to help users debug issues (has been requested in the past, being able to see queries BB is actually making).